packages/flutter/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: A framework for writing Flutter applications
 homepage: https://flutter.dev
 
 environment:
-  sdk: '>=3.2.0-0 <4.0.0'
+  sdk: '>=3.3.0-0 <4.0.0'
```

Comment:
we should make sure this is the same across all pubspec.yaml files the repo

bonus points for writing a test that enforces it

Reply:
I wish I had time for bonus points!